scripts/rtma3d/exrtma3d_gsianl.ksh

```diff
@@ -177,7 +196,7 @@ if [[ ${hrrrmem} -gt 30 ]] && [[ ${HRRRDAS_BEC} -eq 1  ]]; then #if HRRRDAS BEC
   nummem=${hrrrmem}
   cpreq filelist.hrrrdas filelist03
   ${CP} ${PARMgsi}/hybens_info_hrrrdas hybens_info
-  beta1_inv=$(( 1 - $EnsWgt  ))
+  beta1_inv=0.9
```

Comment:
@EdwardColon-NOAA Why did you change beta2_inv to 0.9?It was set to 0.1  (1 - 0.9 = 0.1)

Reply:
Hi Annette. Thank you for the correction. I'll address this now.